libbpf-tools/trace_helpers.c

```diff
@@ -1311,3 +1311,50 @@ int str_to_long(const char *src, void *dest)
 
 	return errno;
 }
+
+int str_loadavg(char *buf, size_t buf_len)
+{
+	int n, err = 0;
+	char avg[64];
+	FILE *f;
+
+	if (!buf || buf_len == 0)
+		return -EINVAL;
+
+	f = fopen("/proc/loadavg", "r");
+	if (!f)
+		return -errno;
+
+	n = fread(avg, 1, sizeof(avg), f);
+	if (!n) {
+		err = -errno;
+		goto cleanup;
+	}
+
+	memset(buf, 0, buf_len);
```

Comment:
memset is unnecessary, as snprintf null-terminates the buffer.
Please refer to the snprintf man page.

Reply:
Fixed, thanks.